design-documents/graph-ql/coverage/CartAddressOperations.graphqls

```diff
@@ -55,30 +62,32 @@ type SetBillingAddressOnCartOutput {
 }
 
 type Cart {
-    addresses: [CartAddress]!
+    shipping_addresses: [ShippingCartAddress]!
+    billing_address: BillingCartAddress!
 }
 
-type CartAddress {
-    firstname: String!
-    lastname: String!
+interface CartAddressInterface {
+    firstname: String
+    lastname: String
     company: String
-    street: [String!]!
-    city: String!
+    street: [String]
+    city: String
     region: CartAddressRegion
     postcode: String
-    country: CartAddressCountry!
-    telephone: String!
-    address_type: AdressTypeEnum!
-
-    selected_shipping_method: CheckoutShippingMethod
-    available_shipping_methods: [CheckoutShippingMethod]!
-
-    items_weight: Float
+    country: CartAddressCountry
+    telephone: String
     customer_notes: String
```

Comment:
I'm not sure if `customer_notes` field makes sense for a billing address.

Reply:
Synchronised with latest version of the schema.
The `customer_notes` is deprecated for Billing Address and removed in current Schema.